src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -384,136 +370,56 @@ def handlePileup(self):
         siteConfig = loadSiteLocalConfig()
         PhEDExNodeName = siteConfig.localStageOut["phedex-node"]
         self.logger.info("Running on site '%s', local PNN: '%s'", siteConfig.siteName, PhEDExNodeName)
+        jsonPileupConfig = os.path.join(self.stepSpace.location, "pileupconf.json")
 
-        pileupDict = self._getPileupConfigFromJson()
-
-        # 2011-02-03 according to the most recent version of instructions, we do
-        # want to differentiate between "MixingModule" and "DataMixingModule"
-        mixModules, dataMixModules = self._getPileupMixingModules()
-
-        # 2011-02-03
-        # on the contrary to the initial instructions (wave), there are
-        # going to be only two types of pileup input datasets: "data" or "mc"
-        # unlike all previous places where pileupType handled in a flexible
-        # way as specified in the configuration passed by the user, here are
-        # the two pileupTypes hardcoded: and we are going to add the "mc"
-        # datasets to "MixingModule"s and only add the "data" datasets to the
-        # "DataMixingModule"s
-
-        # if the user in the configuration specifies different pileup types
-        # than "data" or "mc", the following call will not modify anything
-        self._processPileupMixingModules(pileupDict, PhEDExNodeName, dataMixModules, "data")
```

Comment:
Kenyi, do you know if in the new mode to tweak pileup no longer needs the previous distinction of MixingModule and DataMixingModule (mapped from our workflow attributes MCPileup and DataPileup)?

Honestly, I never understood those well enough. Maybe @klannon and/or @davidlange6 would have some input here(?)

Reply:
`I guess you missed this?` I removed it, but it seems it made it back somehow (during the squash?)
`I guess the answer is no, nothing else is needed, right?` Right.
As for pileup, I think it's still using the same logic here:
https://github.com/cms-sw/cmssw-wm-tools/blob/master/bin/cmssw_handle_pileup.py#L88-L91

Reply:
I definitely did not understand this logic. - no. . I just copied what was in wmcore to preserve functionality.. It seemed overly complicated to me at the time.

Reply:
Thanks for the pointer! I have that repository in my bookmarks now.